src/program/graph.rs

```diff
@@ -21,7 +21,13 @@ use std::fmt;
 use petgraph::graphmap::GraphMap;
 use petgraph::Directed;
 
-use crate::instruction::{FrameIdentifier, Instruction, MemoryReference};
+use crate::instruction::{
```

Comment:
At this point, maybe `use crate::instruction::*`?

Reply:
Not a fan of the implicitness/"magic" of `*`, and since `rust-analyzer` helps maintain the import list I'd say it's not bad